backend/RPI_Smart_Calendar/RPI_Smart_Calendar/settings.py

```diff
@@ -1,7 +1,7 @@
 """
 Django settings for RPI_Smart_Calendar project.
 
-Generated by 'django-admin startproject' using Django 3.2.5.
+Generated by 'django-admin startproject' using Django 3.2.4.
```

Comment:
According to the specification, we should use Django 3.2.5. Please upgrade your Django version.